libs/stringtable/src/analyze/lints/l02_usage/mod.rs

```diff
@@ -0,0 +1,255 @@
+                unused.retain(|s| *s != i_lower);
+                missing.retain(|sp| *sp.0 != i_lower);
+            }
+        }
+        let ignore_missing = config
+            .option("ignore_missing")
+            .is_some_and(|o| o.as_bool().is_some_and(|b| b));
+        let ignore_unused = config
+            .option("ignore_unused")
+            .is_some_and(|o| o.as_bool().is_some_and(|b| b));
+        let ignore_duplicate = config
+            .option("ignore_duplicate")
+            .is_some_and(|o| o.as_bool().is_some_and(|b| b));
+
+        let _ = std::fs::create_dir_all(".hemttout");
+        codes.extend(unused_codes(unused, &all, ignore_unused));
+        codes.extend(missing_codes(&missing, ignore_missing));
+        codes.extend(duplicate_codes(&all, ignore_duplicate));
+        codes
+    }
+}
+
+fn unused_codes(
+    mut unused: Vec<String>,
+    all: &HashMap<String, Vec<crate::Position>>,
+    ignore: bool,
+) -> Codes {
+    let _ = std::fs::remove_file(".hemttout/unused_stringtables.txt");
+    let mut codes: Codes = Vec::new();
+    let _ = std::fs::remove_file(".hemttout/unused_stringtables.txt");
+    if !unused.is_empty() {
+        unused.sort();
+        unused.dedup();
+        let mut file = std::fs::File::create(".hemttout/unused_stringtables.txt")
+            .expect("Failed to create file");
+        for key in &unused {
+            let pos = all
+                .get(key)
+                .expect("unused must exist in all")
+                .first()
+                .expect("must have a position");
+            writeln!(
+                file,
+                "{} - {}:{}:{}",
+                key,
+                pos.path().as_str().trim_start_matches('/'),
+                pos.start().1 .0,
+                pos.start().1 .1
+            )
+            .expect("Failed to write to file");
+        }
+        if !ignore {
+            codes.push(Arc::new(CodeStringtableUnusedFile::new(
+                unused.len() as u64,
+                Severity::Warning,
+            )));
+        }
+    }
+    codes
+}
+
+fn missing_codes(missing: &[(String, Position)], ignore: bool) -> Codes {
+    let _ = std::fs::remove_file(".hemttout/missing_stringtables.txt");
+    let mut codes: Codes = Vec::new();
+    if !missing.is_empty() {
+        let mut file = std::fs::File::create(".hemttout/missing_stringtables.txt")
+            .expect("Failed to create file");
+        for (key, pos) in missing {
+            writeln!(
+                file,
+                "{} - {}:{}:{}",
+                key,
+                pos.path().as_str().trim_start_matches('/'),
+                pos.start().1 .0,
+                pos.start().1 .1
+            )
+            .expect("Failed to write to file");
+        }
+        if !ignore {
+            codes.push(Arc::new(CodeStringtableMissingFile::new(
+                missing.len() as u64,
+                Severity::Error,
```

Comment:
```suggestion
                Severity::Warning,
```
ace does have a false positive on this so it might be better as a warning for a short time?

Reply:
Ah, yeah should be warning overall imo, doesn't need to be an error